angular-client/src/components/battery/battery.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, Input } from '@angular/core';
+import Theme from 'src/services/theme.service';
+
+@Component({
+  selector: 'battery',
+  templateUrl: 'battery.component.html',
+  styleUrls: ['battery.component.css']
+})
+export class BatteryPercentageComponent {
+  @Input() props!: { percentage: number; height: number; width: number };
+  color: string = '';
+  item!: {
```

Comment:
what is item?

Reply:
relic from first design of battery, super unnecessary now, got rid of it